stack/corepersistence/queryindex/src/main/java/org/apache/usergrid/persistence/index/impl/EntityToMapConverter.java

```diff
@@ -137,6 +118,52 @@ else if ( f instanceof UUIDField ) {
         return entityMap;
     }
 
+    private static void putList(Map<String, Object> entityMap, Field field, List list, boolean isRootLevel) {
+        if ( !list.isEmpty() ) {
+            if(isRootLevel) {
```

Comment:
Will this work for nested?  I.E this

{
 "sub1" :{
  "sub2":{ 
   "arrayofstrings": ["foo", "bar"]
  }
}
}